### ISSUE TITLE
Write Word Store model

### ISSUE THREAD

**Issue by dadwalakshay** [dadwalakshay/fourword#6]
Write Django models maintain User's word store.

**Comment by Krishna-Sivakumar** [comment]
Can I work on this?

**Comment by dadwalakshay** [comment]
Yes, but first submit an ER diagram or schema of DB structure. And assume User table is done, am working on it already.

**Comment by Krishna-Sivakumar** [comment]
@dadwalakshay can you provide more details as to what a word store means, and also how it's associated with the user?

**Comment by dadwalakshay** [comment]
Required Django model has been added, please check.
